scripts/windows/version.ps1

```diff
@@ -82,14 +81,14 @@ else {
     $VERSION = $TAG
 }
 
-$env:URI_VERSION = [uri]::EscapeDataString($VERSION)
+$env:URI_VERSION = $VERSION
+# Docker tags cannot include '+'
 $VERSION = $VERSION.Replace('+', '-')
 
 #export stuff out 
 $env:VERSION = $VERSION
 $env:COMMIT = $COMMIT
-$env:REPO = "rancher"
-$env:IMAGE = "$REPO/system-agent-installer-rke2:$VERSION"
+$env:IMAGE = "$env:REPO/system-agent-installer-rke2:$VERSION"
```

Comment:
Can `$env:REPO` ever be unset? Maybe use the `REPO = if () {} else {""}; paradigm instead

Reply:
It is occasionally overridden by devs if they are trying to build a custom image, but even in that case people usually just retag it before pushing it. I suppose we could also see more use of this as we start building and pushing directly to Prime?

Reply:
I've reverted a few of the changes I've made to this file and the `download.ps1` file just to be super sure this won't negatively impact CI. After looking closer I was able to fix the issues I was having when building locally with fewer changes. The CI right now only relies on [`download.ps1`](https://github.com/rancher/system-agent-installer-rke2/blob/7122b8a1d65b81e642a95131fd47683807551fc2/.github/workflows/release.yaml#L94) which relies on `version.ps1`.